tests_backend/tests/updates_test.ts

```diff
@@ -190,6 +190,104 @@ export async function testUpdateEndpoint(backendBaseUrl: URL, supabase: Supabase
     const { error: deleteDeviceError } = await supabase.from('devices').delete().eq('device_id', newDeviceData.device_id)
     assert(deleteDeviceError === null, `Supabase delete device error ${JSON.stringify(deleteDeviceError)} is not null`)
   }
+  await testTwoChannels(backendBaseUrl, supabase)
+}
+
+async function testTwoChannels(_backendBaseUrl: URL, supabase: SupabaseType) {
+  // We update production channel iOS: true then check if two_default channel is still public or not
+  const { error: changeProductiuonChannelError } = await supabase.from('channels').update({ ios: true }).eq('id', 22)
+  assert(changeProductiuonChannelError === null, `Supabase change production channel (ios) error ${JSON.stringify(changeProductiuonChannelError)} is not null`)
+
+  try {
+    await delay(1500)
+    const { data: secondChannel, error: getSecondChannelError } = await supabase
+      .from('channels')
+      .select('*')
+      .eq('id', 24)
+      .single()
+
+    assert(getSecondChannelError === null, `Supabase get second channel error ${JSON.stringify(getSecondChannelError)} is not null`)
+    assert(secondChannel!.public === false, 'Second channel is public')
+  }
+  finally {
+    // Reset to default values
+    const { error: changeProductiuonChannelError2 } = await supabase.from('channels').update({ ios: false }).eq('id', 22)
+    assert(changeProductiuonChannelError2 === null, `Supabase change production channel (ios) error ${JSON.stringify(changeProductiuonChannelError2)} is not null`)
+
+    const { error: changeSecondChannelError } = await supabase.from('channels').update({ public: true }).eq('id', 24)
+    assert(changeSecondChannelError === null, `Supabase change production channel (ios) error ${JSON.stringify(changeSecondChannelError)} is not null`)
```

Comment:
this could be a helper

```suggestion
    await resetChannels()
```

Reply:
@WcaleNieWolny thoughts?

Reply:
This could be a helper, feel free to implement this @kremedev